gui/build.py

```diff
@@ -79,25 +81,37 @@ def do_bundle(options):
     for dirname in ('build', 'dist'):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
-    # create a intaller
-    python_lib_path = os.path.join(options.python_path, "Lib")
-    makensis_path = os.path.join(options.nsis_path, "makensis.exe")
-    gui_path = os.path.realpath('.')
-    moz_path = os.path.realpath('..')
-    python_dir = os.path.join(
-        os.path.dirname(sys.executable),
-        "Scripts"
-    )
-    args = []
-    args.append('--icon=wininst/app_icon.ico')
-    args.append('--base-name=Win32GUI')
-    args.append('--include-path=%s;%s;%s\site-packages;%s'
-                % (gui_path, moz_path, python_lib_path, python_lib_path))
-    args.append('--target-name=mozregression-gui.exe')
-    args.append('--target-dir=dist')
-    args.append('mozregui/main.py')
-    call(sys.executable, os.path.join(python_dir, 'cxfreeze'), *args)
-    call(makensis_path, 'wininst.nsi', cwd='wininst')
+    # create a installer
+    if sys.platform == 'win32':
+        python_lib_path = os.path.join(options.python_path, "Lib")
+        makensis_path = os.path.join(options.nsis_path, "makensis.exe")
+        gui_path = os.path.realpath('.')
+        moz_path = os.path.realpath('..')
+        python_dir = os.path.join(
+            os.path.dirname(sys.executable),
+            "Scripts"
+        )
+        args = []
+        args.append('--icon=wininst/app_icon.ico')
+        args.append('--base-name=Win32GUI')
+        args.append('--include-path=%s;%s;%s\site-packages;%s'
+                    % (gui_path, moz_path, python_lib_path, python_lib_path))
+        args.append('--target-name=mozregression-gui.exe')
+        args.append('--target-dir=dist')
+        args.append('mozregui/main.py')
+        call(sys.executable, os.path.join(python_dir, 'cxfreeze'), *args)
+        call(makensis_path, 'wininst.nsi', cwd='wininst')
+    elif sys.platform == 'linux2':
+        python_lib_path = options.python_path
+        gui_path = os.path.realpath('.')
+        moz_path = os.path.realpath('..')
+        os.system('cxfreeze --include-path="%s:%s:%s:%s/dist-packages"'
```

Comment:
You should use our call() function instead of os.system, because it is more robust for arguments that could contains spaces, and the exit code of the subprocess is tested.